webdriver/tests/get_window_rect.py

```diff
@@ -151,11 +151,10 @@ def test_payload(session):
     assert response.status == 200
     assert isinstance(response.body["value"], dict)
     value = response.body["value"]
-    assert "width" in value
-    assert "height" in value
-    assert "x" in value
-    assert "y" in value
-    assert isinstance(value["width"], int)
-    assert isinstance(value["height"], int)
-    assert isinstance(value["x"], int)
-    assert isinstance(value["y"], int)
```

Comment:
I’m not entirely comfortable removing the explicit assertions of the data integrity, but I guess this is fine.

Reply:
In case `window.screenX` doesn't return an integer, for example?

Reply:
Yes, this has known to happen. But I think pytest will catch this when it does a deep dict comparison?

Reply:
Right, it does compare the entire dict. Obviously, if `window.screenX` and what Get Window Rect return are broken in the same way then it won't catch it now. So maybe we should have these?

Reply:
That would be out of scope of the WebDriver tests, but I wouldn’t object to keeping the assertions around.

Reply:
Just to give some context: Marionette used to return floats for these values.

Reply:
But if it did, then it wouldn't match `window.screenX`, no? (Assuming a conforming `window.screenX` implementation.)

Reply:
Yes, I would assume so. As I said, happy to land this as-is.